.travis.yml

```diff
@@ -21,9 +14,8 @@ script:
   - docker run my-test-image tests/test_integration.py
   - >
      docker run
-     -e API_KEY=$API_KEY
-     -e TOKEN=$TOKEN
-     -e BOARD_ID=$BOARD_ID
+     -e MONGO_DB_CONNECTION=$MONGO_DB_CONNECTION
```

Comment:
Here you're setting the non-cosmos version of these variables; that doesn't feel right given the app uses the cosmos variables?